### ISSUE TITLE
tokio-tls 0.2.1 MSRV raised to 1.28.0 by pkg-config, v0.1.x CI broken

### ISSUE THREAD

**Issue by dekellum** [tokio-rs/tokio#1369]
## Version

tokio-tls 0.2.1

## Platform

x86_64-unknown-linux-gnu

## Subcrates

tokio-tls

## Description

MSRV tested and advertised for current tokio-tls 0.2.1 release and current v0.1.x branch is 1.26.0. However [pkg-config 0.3.15 was recently released with MSRV raised to 1.28.0](https://github.com/rust-lang/pkg-config-rs/blob/master/CHANGELOG.md#0315---2019-07-25) (rust-lang/pkg-config-rs#76). Yes, this is an MSRV bump in a PATCH release, but Tokio isn't in much position to complain about that.

Here is the dependency tree:

``` txt
pkg-config v0.3.15
[build-dependencies]
└── openssl-sys v0.9.48
    ├── native-tls v0.2.3
    │   └── tokio-tls v0.2.1 (/home/david/src/tokio/tokio-tls)
    └── openssl v0.10.24
        └── native-tls v0.2.3 (*)
        [dev-dependencies]
        └── tokio-tls v0.2.1 (/home/david/src/tokio/tokio-tls) (*)
```

In v0.1.x CI of MSRV 1.26.0 (via `cargo check --all`) this leads to the following (originally found via #1368): 

https://dev.azure.com/tokio-rs/Tokio/_build/results?buildId=1705

``` txt
  Compiling pkg-config v0.3.15
error[E0658]: use of unstable library feature 'collections_range': might be replaced with `Into<_>` and a type containing two `Bound` values (see issue #30877)
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/pkg-config-0.3.15/src/lib.rs:75:23
   |
75 | use std::ops::{Bound, RangeBounds};
   |                       ^^^^^^^^^^^

error[E0658]: use of unstable library feature 'collections_range': might be replaced with `Into<_>` and a type containing two `Bound` values (see issue #30877)
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/pkg-config-0.3.15/src/lib.rs:244:12
    |
244 |         R: RangeBounds<&'a str>
    |            ^^^^^^^^^^^^^^^^^^^^

error[E0599]: no method named `start_bound` found for type `R` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/pkg-config-0.3.15/src/lib.rs:246:40
    |
246 |         self.min_version = match range.start_bound() {
    |                                        ^^^^^^^^^^^

error[E0599]: no method named `end_bound` found for type `R` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/pkg-config-0.3.15/src/lib.rs:251:40
    |
251 |         self.max_version = match range.end_bound() {
    |                                        ^^^^^^^^^

error: aborting due to 4 previous errors
```

**Comment by carllerche** [comment]
We will have to pump ours.

**Comment by dekellum** [comment]
The CI part is now fixed by #1451.  Closing this, but feel free to reopen if there is anything more to do with it?
